docs/source/_templates/index.html

```diff
@@ -1,92 +0,0 @@
-{% extends "layout.html" %}
-{% set title = _('Overview') %}
-{% block body %}
-  <h1>{{ _('Welcome') }}</h1>
-
-
-  <h2 style="margin-bottom: 10 px">{%trans%}Start Here{%endtrans%}</h2>
-
-  <table class="contentstable" align="center" style="margin-left: 30px">
```

Comment:
it uses table for a two column while css have ability to create multiple column, so let's use it. (see new file) 
Moreover this allow to dynamically change the number of columns depending on device width.